FNA.csproj

```
@@ -402,6 +402,7 @@
     <Compile Include="src\Utilities\FileHelpers.cs" />
     <Compile Include="src\Utilities\FNADllMap.cs" />
     <Compile Include="src\Utilities\FNAInternalExtensions.cs" />
+    <Compile Include="src\Utitlies\MarshalHelper.cs" />
```

```suggestion
    <Compile Include="src\Utilities\MarshalHelper.cs" />
```
This should probably be done in Makefile as well.

---

Fixed